fastcov.py

```diff
@@ -196,9 +194,11 @@ def exclMarkerWorker(fastcov_sources, chunk, exclude_branches_sw):
         end_line = 0
         with open(source) as f:
             for i, line in enumerate(f, 1): #Start enumeration at line 1
-                # Exclude branches starting with...
-                for e in exclude_branches_sw:
-                    if line.lstrip().startswith(e) and str(i) in fastcov_sources[source]["branches"]:
+                if str(i) in fastcov_sources[source]["branches"]:
+                    if all(not line.lstrip().startswith(e) for e in include_branches_sw): # Include branches starting with...
```

Comment:
This should maybe only be done if there is a whitelist in the first place?

Reply:
Oh, I see what you are saying now -

Should be `if include_branches_sw and all(...)`

This is quite possibly a bug, let me run a few tests.

Reply:
You're right, I need to setup Travis CI to prevent regressions in the future!

Reply:
Ok, cool - I wasn't sure if I only misunderstood the new feature in the beginning.. ;)

I also suggest using pytest then.. :)
How do you run the existing test?

Reply:
Right now I run existing unit tests (of which there are not many) manually. I also manually run fastcov against nlohmann/json to make sure coverage hasn't changed.

I plan on adding black box testing that actually runs fastcov against CMake projects with predetermined coverage and branching.

Please check out #16 to see if it resolves your issues (I will test a bit more before merging).

Reply:
Yes, works.